recipes/compute_library/all/conanfile.py

```diff
@@ -0,0 +1,126 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, rm, chdir
+from conan.tools.build import check_min_cppstd, cross_building, build_jobs
+from conan.tools.scm import Version
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.gnu import AutotoolsDeps
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class ComputeLibraryConan(ConanFile):
+    name = "compute_library"
+    description = "The Compute Library is a set of computer vision and machine learning functions optimized for both Arm CPUs and GPUs using SIMD technologies"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ARM-software/ComputeLibrary"
+    topics = ("android", "linux", "machine-learning", "arm", "computer-vision", "neural-network")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "enable_openmp": [True, False],
+        "enable_opencl": [True, False],
+        "enable_neon": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "enable_openmp": False,
+        "enable_opencl": True,
+        "enable_neon": True,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 14
```

Comment:
```
2023-09-19T22:49:09.7725782Z ======== Input profiles ========
2023-09-19T22:49:09.7725963Z Profile host:
2023-09-19T22:49:09.7726141Z [settings]
2023-09-19T22:49:09.7726374Z arch=x86_64
2023-09-19T22:49:09.7726707Z build_type=Release
2023-09-19T22:49:09.7727039Z compiler=gcc
2023-09-19T22:49:09.7727365Z compiler.cppstd=gnu14
2023-09-19T22:49:09.7727745Z compiler.libcxx=libstdc++11
2023-09-19T22:49:09.7728025Z compiler.version=9
2023-09-19T22:49:09.7728170Z os=Linux
2023-09-19T22:49:09.7729469Z 
2023-09-19T22:49:09.7733545Z Profile build:
2023-09-19T22:49:09.7733805Z [settings]
2023-09-19T22:49:09.7734048Z arch=x86_64
2023-09-19T22:49:09.7734584Z build_type=Release
2023-09-19T22:49:09.7734751Z compiler=gcc
2023-09-19T22:49:09.7734943Z compiler.cppstd=gnu14
2023-09-19T22:49:09.7735107Z compiler.libcxx=libstdc++11
2023-09-19T22:49:09.7735283Z compiler.version=9
2023-09-19T22:49:09.7735441Z os=Linux
2023-09-19T22:49:09.7735501Z 
2023-09-19T22:49:09.7747813Z 
2023-09-19T22:49:09.7748454Z ======== Computing dependency graph ========
2023-09-19T22:49:09.7809557Z opencl-headers/2023.04.17: Not found in local cache, looking in remotes...
2023-09-19T22:49:09.7809956Z opencl-headers/2023.04.17: Checking remote: conancenter
2023-09-19T22:49:10.1176262Z opencl-headers/2023.04.17: Downloaded recipe revision 3d98f2d12a67c2400de6f11d5335b5a6
2023-09-19T22:49:10.1212693Z scons/4.3.0: Not found in local cache, looking in remotes...
2023-09-19T22:49:10.1213163Z scons/4.3.0: Checking remote: conancenter
2023-09-19T22:49:10.5420139Z scons/4.3.0: Downloaded recipe revision 3476cd7ed0d4f2913159fb38dd8d8b7d
2023-09-19T22:49:10.5463281Z Graph root
2023-09-19T22:49:10.5463733Z     cli
2023-09-19T22:49:10.5464354Z Requirements
2023-09-19T22:49:10.5465147Z     compute_library/23.08#6942370724cbbaf7848925eef9ffdf0c - Cache
2023-09-19T22:49:10.5465493Z     opencl-headers/2023.04.17#3d98f2d12a67c2400de6f11d5335b5a6 - Downloaded (conancenter)
2023-09-19T22:49:10.5465671Z Build requirements
2023-09-19T22:49:10.5465961Z     scons/4.3.0#3476cd7ed0d4f2913159fb38dd8d8b7d - Downloaded (conancenter)
2023-09-19T22:49:10.5466058Z 
2023-09-19T22:49:10.5466204Z ======== Computing necessary packages ========
2023-09-19T22:49:10.7981881Z Requirements
2023-09-19T22:49:10.7982764Z     compute_library/23.08#6942370724cbbaf7848925eef9ffdf0c:e1c99526e572359ac75605c4d1fc6b82784c0857 - Invalid
2023-09-19T22:49:10.7983027Z Build requirements
2023-09-19T22:49:10.7983194Z Skipped binaries
2023-09-19T22:49:10.7983474Z     opencl-headers/2023.04.17, scons/4.3.0
2023-09-19T22:49:10.8001267Z 
2023-09-19T22:49:10.8002089Z ======== Installing packages ========
2023-09-19T22:49:10.8004392Z ERROR: There are invalid packages:
2023-09-19T22:49:10.8004783Z compute_library/23.08: Invalid: compute_library/23.08 requires C++14, which your compiler does not support.
```
Why do we have an exception, while `compiler.cppstd=gnu14` ?

Reply:
it was because was requesting GCC 10 as minimum version, but you are using GCC 9. It's fixed now to request GCC 6, due C++14 support.